spec/controllers/untrusted_examples.rb

```diff
@@ -1,9 +1,6 @@
 RSpec.shared_examples 'disallows untrusted ips' do
   context 'an untrusted ip' do
-    before do
-      allow_any_instance_of(ActionDispatch::Request).to receive(:remote_ip).
-        and_return('192.168.1.0')
-    end
+    before do request.headers['REMOTE_ADDR'] = '192.168.1.0' end
```

Comment:
This looks like a rubocop autocorrection, prefer to stick to the general convention of multi-line  for `do/end` blocks